app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -59,7 +59,7 @@
   <%= render "govuk_publishing_components/components/inset_text" do %>
     <p class="govuk-body">The 2i reviewer is <%= @step_by_step_page.reviewer.name %></p>
     <% if current_user.permissions.include?("2i reviewer") %>
-      <%= link_to "The 2i reviewer is #{@step_by_step_page.reviewer.name}. See guidance for doing 2i review.", step_by_step_page_guidance_path(@step_by_step_page) %>
+      <%= link_to "See guidance for doing 2i review.", step_by_step_page_guidance_path(@step_by_step_page) %>
```

Comment:
could you wrap this into a paragraph and add a `govuk-link` class to it
`<%= tag.p link_to("See guidance for doing 2i review.", step_by_step_page_guidance_path(@step_by_step_page), class: "govuk-link"), class: "govuk-body" %>`